WoWsShipBuilder.Web/WoWsShipBuilder.Web.csproj

```diff
@@ -75,4 +74,9 @@
       <AvaloniaResource Include="wwwroot\assets\nation_flags\flag_Usa.png" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
Remove this ItemGroup entirely

Reply:
This is not related to this pull request, it probably got picked up from the blazor web app branch when i created this branch